scripts/UI/Styles/ThemeColors.reds

```diff
@@ -2,7 +2,8 @@
 // Codeware.UI.ThemeColors
 // -----------------------------------------------------------------------------
 //
-// - Reference for base colors (should be replaced by ThemePicker)
+// Deprecated: you should use BindProperty and main_colors.inkstyle to use 
```

Comment:
I wouldn't mark it deprecated until `ThemePicker`/`StylePicker` is implemented. Style bindings can't be used with animations. For animations you need raw values which ideally should be read from style.

Reply:
Alright, hence the pickers, now I understand. I'll change it.